Valencia/WorldClockViewController.swift

```diff
@@ -18,6 +18,7 @@ class WorldClockViewController: UIViewController, UITableViewDelegate, UITableVi
   }
 
   var selectedCityInWorldClock: [String] = []
+  var date = Date()
```

Comment:
we don't need this as a property for now.

Reply:
I fixed it